_episodes/03-multiple_files.md

```diff
@@ -73,15 +73,14 @@ print(filenames)
 This will give us a list of all the files which end in `*.out` in the `outfiles` directory. Now if we want to parse every file we just read in, we will use a `for` loop to go through each file.
 ```
 for f in filenames:
-    outfile = open(f,'r')
-    data = outfile.readlines()
-    outfile.close()
-    for line in data:
-        if 'Final Energy' in line:
-            energy_line = line
-            words = energy_line.split()
-            energy = float(words[3])
-            print(energy)
+    with open(f,'r') as data:
+        for line in data:
+            if 'Final Energy' in line:
+                energy_line = line
+                words = energy_line.split()
+                energy = float(words[3])
+                print(energy)
+                break
```

Comment:
Please remove the break statement. We do not wish to introduce this concept.

Reply:
I have remove all `break` related line in this PR!